openvpn-install.sh

```diff
@@ -264,10 +282,25 @@ else
 		# Ubuntu >= 16.04 and Debian > 8 have OpenVPN > 2.3.3 without the need of a third party repository.
 		# The we install OpenVPN
 		apt-get install openvpn iptables openssl wget ca-certificates curl -y
-	else
-		# Else, the distro is CentOS
+	elif [[ "$OS" = 'centos' ]]; then
 		yum install epel-release -y
 		yum install openvpn iptables openssl wget ca-certificates curl -y
+	else
+		# Else, the distro is ArchLinux
+		echo ""
+		echo ""
+		echo "As you're using ArchLinux, I need to update the packages on your system to install whose I need."
```

Comment:
With APT, when a package requires dependencies, and those dependencies are installed and have updates availble, they're updated when the package is installed. In the same way, when an update for the installed package `example` is available, doing `apt install example` will just update the package.

I assume it works the same way for pacman ? Correct me if I'm wrong. 

Thus, I don't think that adding `-u` to the pacman command is needed. Not that this is not a good idea, but I don't want the script to be too intrusive for the user. The `-y` is a good idea, though.

I suggest the removal of lines 290 to 302. What do you think ? :)